addons/hr_holidays/static/src/core/common/@types/models.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "models" {
+    export interface ResUsers {
+        leave_date_to: string;
```

Comment:
Already the case before this PR, but we should probably define `leave_date_to` as a datetime field.

Reply:
I didn't change it because the only use case is passing it to `getOutOfOfficeDateEndText` which expects a string.